iam/hooks.py

```diff
@@ -285,6 +288,36 @@ def is_unverified_user(cog, obj, *func_args, **func_kwargs):
         return False, "You are already verified."
     return True, None
 
+def is_strictly_verified_user(cog, obj, *func_args, **func_kwargs):
```

Comment:
What is a "strictly verified user"?

Reply:
A user who has a verified entry in the database, and not just the role in the server.